utils/include/CConfiguration.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "IParser.h"
+#include "SConfigurationStructures.h"
+
+class CConfiguration
+{
+public:
+	CConfiguration() = delete;
+	CConfiguration(CConfiguration&) = delete;
+	CConfiguration(CConfiguration&&) = delete;
+
+
+
+	explicit CConfiguration(std::unique_ptr<IParser> p_parser);
+
+	SServer& GetDefaultServerConfiguration();
+	SComunicationSettings& GetDefaultComunicationConfiguration();
+	SLogging& GetDefaultLoggingConfiguration();
+	STime& GetDefaultTimeConfiguration();
+	SThreadPool& GetDefaultThreadPoolConfiguration();
+	SHDDInfo& GetDefaultHDDInfoConfiguration();
+	SProcessesInfo& GetDefaultProcessInfoConfiguration();
+
+	SServer& TryToGetServerConfigurationFromFile();
```

Comment:
Instead of TryToGet<NAME> functions better option is to use 1 initialize method where you read configuration from file and it in structures.

Reply:
Do i need to implement selective config reading too, in case if we would want to read only one non-standart setting? for example(Service settings only)